webapp/channels/src/plugins/export.js

```diff
@@ -91,6 +92,7 @@ window.Components = {
     ThreadViewer,
     CreatePost,
     PostMessagePreview,
+    AdvanceTextEditor,
```

Comment:
There's a small typo here. This should be `AdvancedTextEditor`

Reply:
@crspeller If we're switching the AI plugin to use `AdvancedTextEditor` instead of `CreatePost`, do you think we could remove `CreatePost` from these exports? The AI plugin should be the only one using this, so we should remove it if it's not needed to minimize the surface area that we need to check for backwards compatibility on.

Given the plugin is experimental, I think it would be reasonable for force people to say that the new version of MM requires the newest version of the plugin. This'll also be a new version of the plugin only works on the latest version of MM. This'll also be going in for 10.0, so it's a good time to break something if we have to

Reply:
@hmhealey I don't mind saying that v10 requires 1.0. I would like to preserve that 1.0 does not require v10 and that shouldn't be a problem.

Reply:
If I understand correctly, that means that we could remove the `CreatePost` export and the plugin would have to support either `CreatePost` or `AdvancedTextEditor` for the time being based on the server version. Is that right?

I'm good with that approach, although I do wonder if we might want to use 1.0 as a reason to encourage people to upgrade to MM 10.0. There was a while where we were focused on trying to encourage people to upgrade, and it could be a way to encourage that, but I don't know how much that's a priority for the team at the moment